docs/feed.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: sub-navigation
+order: 7
+title: Adding a feed
+description: Allow people to subscribe to your posts and read them in a feed reader
+---
+
+You can add a feed using the [XML Atom format](<https://en.wikipedia.org/wiki/Atom_(web_standard)>) by following these instructions.
+
+## Create a feed
+
+To create a feed, add a file named `feed.njk` with the following content:
+
+```yaml
+---
+eleventyExcludeFromCollections: true
+layout: feed
+permalink: /feed.xml
+---
+```
+
+The `permalink` value is the location of the generated feed.
+
+## Create a collection of posts
+
+The feed will include all pages in a collection called 'post'.
+
+You can create this by adding some code to your `eleventy.config.js`:
+
+```js
+eleventyConfig.addCollection('post', (collection) => {
+  return collection.getFilteredByGlob('app/posts/*.md')
+})
+```
+
+## Add the URL of your feed to your options
+
+To make the feed discoverable, add the full URL of your feed as `feedUrl` within the options for this plugin in your `.eleventy.config.js` options file:
+
+```js
+const govukEleventyPlugin = require('@x-govuk/govuk-eleventy-plugin')
+
+eleventyConfig.addPlugin(govukEleventyPlugin, {
+  feedUrl: 'https://your-website.example/feed.xml'
```

Comment:
Given the use of the `canonicalUrl` filter, I think you can use this instead (saves issues with GitHub Pages URLs possibly being in sub-folder, etc.) 

```suggestion
  feedUrl: 'feed.xml'
```